sdformat-config.cmake.in

```diff
@@ -28,4 +29,4 @@ endforeach()
 
 # Add a root gz::lib alias
 add_library(@LIB_NAME_COMP_PREFIX@ INTERFACE IMPORTED)
-target_link_libraries(@LIB_NAME_COMP_PREFIX@ INTERFACE @LIB_NAME_COMP_PREFIX@::core ${ALL_COMPONENTS})
+target_link_libraries(@LIB_NAME_COMP_PREFIX@ INTERFACE ${ALL_COMPONENTS})
```

Comment:
As we discussed offline, it probably makes more sense for this to map to `@LIB_NAME_COMP_PREFIX@::@LIB_NAME_COMP_PREFIX@` to be consistent with cmake and gz-cmake conventions.

Reply:
Done in https://github.com/gazebo-release/sdformat_vendor/pull/2/commits/5d7b826073e43910781b59d621eca59a28a580cc. Also removed `ALL_COMPONENTS` since it's no longer needed. 

Also, I'm not sure if the `-all` target works. Don't we need to create an alias for it?

Reply:
You mean that we don't have it implemented in gz-cmake or here?

Reply:
We don't have it implemented here. You can do
```cmake
find_package(gz-common5 COMPONENTS all)

target_link_libraries(my_library gz-common5::all)
```
But I don't think that works with our shim here. Currently, you should be able to do `find_package(gz-common COMPONENTS all)`, but not `target_link_libraries(my_library gz-common::all)` (these are without version number)

Reply:
Okay, that was as simple as removing the lines that skipped the "all" target (https://github.com/gazebo-release/sdformat_vendor/pull/2/commits/d62f713a48a7ebf363453951575f38829e946f93). `target_link_libraries(my_library sdformat::all)` should now work.